src/main/java/ua/danit/final_project/services/crud/EmployeeService.java

```diff
@@ -1,7 +1,12 @@
 package ua.danit.final_project.services.crud;
 
 
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.springframework.transaction.annotation.Transactional'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=118&issues=AWXyhLRgxJR_uy9BLCPN&open=AWXyhLRgxJR_uy9BLCPN)

[ISSUE_KEY]: AWXyhLRgxJR_uy9BLCPN